### DISCUSSION TITLE
Tidak bisa publish shiny

### DISCUSSION THREAD

**Discussion by ridhwan4** [teamalgoritma/community#419] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/72968369/198334898-493923be-e8c4-4a29-99e7-e724cfdcce62.PNG)

Halo Kak/Mas izin bertanya kalau shiny nya error tidak bisa dipublish kenapa ya ?

**Comment by VictorNugraha** [comment, marked as answer]
Hai, @ridhwan4 !

Untuk melihat error yang terjadi, bisa menunjungi bagian logs pada wesbite shinyapps.io. Berikut cara menuju bagian logs,

1. Bukalah bagian Dashboard, lalu pilihah app Shiny yang telah Ibu upload. Sebagai contoh, nama applikasi saya InvestNow.
![image](https://user-images.githubusercontent.com/28379954/198822361-ed9db46a-a644-4b5e-84eb-45631d18cb8a.png)

2. Setelah masuk, bisa memilh bagian logs

![image](https://user-images.githubusercontent.com/28379954/198822369-2caa5fbf-c076-427e-b652-c8ba0497c186.png)

3. Nantinya dibagian logs bisa melihat informasi error yang terjadi.

![image](https://user-images.githubusercontent.com/28379954/198822382-d23ef09d-053f-404a-a9b2-e2ac676eb015.png)

Demikian infromasi yang dapat saya sampaikan. Semoga membantu.
